File: documentation/user/en/query/filtering/examples/comparable/attribute-greater-than.java
final EvitaResponse<SealedEntity> entities = evita.queryCatalog(
	"evita",
	session -> {
		return session.querySealedEntity(
			query(
				collection("Product"),
				filterBy(
					attributeGreaterThan("battery-life", "40")
				),
				require(
					entityFetch(
						attributeContent("code", "battery-life")
					)
				)
			)
		);
	}
);

File: documentation/user/en/query/filtering/examples/comparable/attribute-greater-than.cs
EvitaResponse<ISealedEntity> entities = evita.QueryCatalog(
	"evita",
	session => session.QuerySealedEntity(
        Query(
        	Collection("Product"),
        	FilterBy(
        		AttributeGreaterThan("battery-life", "40")
        	),
        	Require(
        		EntityFetch(
        			AttributeContent("code", "battery-life")
        		)
        	)
        )
	)
);